### PR TITLE
Update the code owner of `feeds` service from  FMS group to OPCORE

### PR DESCRIPTION
**Pull request by ro-tex**
The `feeds` service has [FMS](https://github.com/orgs/smartcontractkit/teams/fms) team as its code owner but that group is deprecated and replaced by the [OPCORE](https://github.com/orgs/smartcontractkit/teams/op-core) team. This PR updates the ownership.

Related ticket: [OPCORE-916](https://smartcontract-it.atlassian.net/browse/OPCORE-916)

### Resolves Dependencies

[OPCORE-916]: https://smartcontract-it.atlassian.net/browse/OPCORE-916?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by graham-chainlink — APPROVED** [review]

**Review by ilija42 — APPROVED** [review]

**Review by jmank88 — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 # Services
 /core/services/directrequest @smartcontractkit/keepers
-/core/services/feeds @smartcontractkit/FMS
+/core/services/feeds @smartcontractkit/op-core @eutopian @yevshev
```
Comment (patrickhuie19) [inline .github/CODEOWNERS:14]: When possible we want to use team based owners over individual contributors

**Review by patrickhuie19 — DISMISSED** [review]
When possible we want to use team based owners over individual contributors. We can make exceptions for efforts that don't yet have the maintenance of a full team to support or never will. AFAIK feeds does not fit in this category.

File: .github/CODEOWNERS (hunk 1 above)
Comment (jmank88) [reply .github/CODEOWNERS:14]: I thought the rule was to require a team, not to disallow individuals?

Comment (patrickhuie19) [reply .github/CODEOWNERS:14]: Individuals are not disallowed entirely, but they are exceptions

Comment (ro-tex) [reply .github/CODEOWNERS:14]: These are definitely exceptions for us as well - they are no longer part of OPCore but their expertise in the product (CLO) makes them extremely valuable contributors which we continue to rely on.

**Review by eutopian — APPROVED** [review]

**Pull request merged**